airbyte-commons-converters/src/main/java/io/airbyte/converters/CatalogClientConverters.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
  */
 
-package io.airbyte.workers.helper;
+package io.airbyte.converters;
```

Comment:
Nit: All the other commons have `commons` in their package names, should we keep this pattern for this new module?

Reply:
Probably.  I will do a rename.